src/components/layout/app/Content.tsx

```diff
@@ -7,7 +7,14 @@ const AppLayoutContent: ComponentWithAs<'div', GridItemProps> = forwardRef<
   GridItemProps,
   'div'
 >((props, ref) => {
-  return <GridItem area="content" ref={ref} {...(props as GridItemProps)} />;
+  return (
+    <GridItem
+      area="content"
+      ref={ref}
+      {...(props as GridItemProps)}
+      marginBottom={{ md: '4xl', base: '3xl' }}
```

Comment:
This will affect every single page, also in seller-web. Did you check that?
I am ok with that change but I would consider removing the padding from the `BaseLayout`. Otherwise it will add up margin and padding from two different components and may create confusion. Wdyt?

Reply:
Yea, about that...I had couple of different iterations with Sancho about this, so I in the end I did calculate the paddings on the `baseLayout` to put `margin-bottom` only on content, because the idea was that we want that spacing on every single page. And this seemed like the most straightforward approach without fiddling too much with the rest of the components.

The reason I didn't want to remove padding is then we would only remove `padding-bottom` from `baseLayout` and also we should do the same with the `<aside>` so wanted to leave it as it as and just add on top of it.

To be honest not sure what would be the best approach. Maybe completely remove the padding on `BaseLayout` and add that padding or margin to `Content` then?

Reply:
yes I agree. If you checked with sancho and seller team, I'd consider removing the padding on the base layout and use the margin from here.
Maybe you can open a PR in both projects with the pre-release to see how it looks like